Product.html

```diff
@@ -0,0 +1,283 @@
+                            <i class="bi bi-star-fill"></i>
+                            <i class="bi bi-star-fill"></i>
+                            <i class="bi bi-star-fill"></i>
+                            <i class="bi bi-star-half"></i>
+                    </div>
+                    <p>$50.00</p>
+                </div>
+            </div>
+            <div class="col-4">
+                <img src="images/product-5.jpg" alt="product-5">
+                <h4>Red Printed Tshirt</h4>
+                <div class="rating">
+                    <div class="rating">
+                        <i class="bi bi-star-fill"></i>
+                            <i class="bi bi-star-fill"></i>
+                            <i class="bi bi-star-fill"></i>
+                            <i class="bi bi-star-fill"></i>
+                            <i class="bi bi-star-half"></i>
+                    </div>
+                    <p>$50.00</p>
+                </div>
+            </div>
+            <div class="col-4">
+                <img src="images/product-6.jpg" alt="product-6">
+                <h4>Red Printed Tshirt</h4>
+                <div class="rating">
+                    <div class="rating">
+                        <i class="bi bi-star-fill"></i>
+                            <i class="bi bi-star-fill"></i>
+                            <i class="bi bi-star-fill"></i>
+                            <i class="bi bi-star-fill"></i>
+                            <i class="bi bi-star-half"></i>
+                    </div>
+                    <p>$50.00</p>
+                </div>
+            </div>    
+            <div class="col-4">
+                <img src="images/product-7.jpg" alt="product-7">
+                <h4>Red Printed Tshirt</h4>
+                <div class="rating">
+                    <div class="rating">
+                        <i class="bi bi-star-fill"></i>
+                            <i class="bi bi-star-fill"></i>
+                            <i class="bi bi-star-fill"></i>
+                            <i class="bi bi-star-fill"></i>
+                            <i class="bi bi-star-half"></i>
+                    </div>
+                    <p>$50.00</p>
+                </div>
+            </div>    
+            <div class="col-4">
+                <img src="images/product-8.jpg" alt="product-8">
+                <h4>Red Printed Tshirt</h4>
+                <div class="rating">
+                    <div class="rating">
+                        <i class="bi bi-star-fill"></i>
+                            <i class="bi bi-star-fill"></i>
+                            <i class="bi bi-star-fill"></i>
+                            <i class="bi bi-star-fill"></i>
+                            <i class="bi bi-star-half"></i>
+                    </div>
+                    <p>$50.00</p>
+                </div>
+            </div>
+            <div class="col-4">
+                <img src="images/product-9.jpg" alt="product-9">
```

Comment:
Place proper alt text since `alt="product-9"` doesn't specify the description about image